<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RVIOscilloscope</class>
 <widget class="QMainWindow" name="RVIOscilloscope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>342</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RVI Osciloscope</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="mainLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSaveImage"/>
    <addaction name="actionSaveData"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionDockControls"/>
    <addaction name="actionShowControls"/>
    <addaction name="separator"/>
    <addaction name="actionBackgroundColor"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuVer"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainTB">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPause"/>
   <addaction name="actionZoom"/>
   <addaction name="actionEnChAGrid"/>
   <addaction name="actionEnTimeGrid"/>
   <addaction name="actionEnChBGrid"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="controlsDock">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string comment="controls">Controls</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="controlsWidget">
    <layout class="QVBoxLayout" name="toolsLayout">
     <property name="margin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>&amp;Manual</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rviboardtools.qrc">
     <normaloff>:/images/media-playback-pause.png</normaloff>:/images/media-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause the plot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionShowControls">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show controls</string>
   </property>
   <property name="toolTip">
    <string>Show board contros </string>
   </property>
  </action>
  <action name="actionBackgroundColor">
   <property name="icon">
    <iconset resource="rviboardtools.qrc">
     <normaloff>:/images/fill-color.png</normaloff>:/images/fill-color.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Background color...</string>
   </property>
   <property name="toolTip">
    <string>Set plot background color...</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/zoom-in.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="toolTip">
    <string>Zoom area from plot</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="icon">
    <iconset resource="rviboardtools.qrc">
     <normaloff>:/images/document-save-image.PNG</normaloff>:/images/document-save-image.PNG</iconset>
   </property>
   <property name="text">
    <string>Save &amp;image...</string>
   </property>
   <property name="toolTip">
    <string>Save an image of the current plot...</string>
   </property>
  </action>
  <action name="actionDockControls">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Dock controls</string>
   </property>
   <property name="toolTip">
    <string>Dock controls in the main wondows</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="icon">
    <iconset resource="rviboardtools.qrc">
     <normaloff>:/images/document-save.png</normaloff>:/images/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save data...</string>
   </property>
   <property name="toolTip">
    <string>Save the current plot data</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionEnChAGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/grid_left.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show channel A grid</string>
   </property>
   <property name="toolTip">
    <string>Enable/disable channel A grid in plotter</string>
   </property>
  </action>
  <action name="actionEnChBGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rviboardtools.qrc">
     <normaloff>:/images/grid_right.png</normaloff>
     <normalon>:/images/grid_right.png</normalon>:/images/grid_right.png</iconset>
   </property>
   <property name="text">
    <string>Show channel B grid</string>
   </property>
   <property name="toolTip">
    <string>Enable/disable channel B grid in plotter</string>
   </property>
  </action>
  <action name="actionEnTimeGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rviboardtools.qrc">
     <normaloff>:/images/grid_time.png</normaloff>
     <normalon>:/images/grid_time.png</normalon>:/images/grid_time.png</iconset>
   </property>
   <property name="text">
    <string>Show time grid</string>
   </property>
   <property name="toolTip">
    <string>Enable/disable channel A grid in plotter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rviboardtools.qrc"/>
 </resources>
 <connections/>
</ui>
